src/slack_api_util.ts

```diff
@@ -288,6 +288,20 @@ export async function updateSlackChannelNamesAndIdsInRedis(
       slackChannelNamesAndIds
     );
   }
+
+  return slackChannelNamesAndIds;
+}
+
+// Get Slack Channel Name / ID map, using cache if needed
+export async function getSlackChannelNamesAndIds(
+  redisClient: PromisifiedRedisClient
+): Promise<SlackChannelNamesAndIds | null> {
+  const slackChannelIds = await RedisApiUtil.getHash(
+    redisClient,
+    'slackPodChannelIds'
+  );
+  if (slackChannelIds) return slackChannelIds;
+  return updateSlackChannelNamesAndIdsInRedis(redisClient);
```

Comment:
Helper function to query API directly if cache isn't set. Might be nice to use elsewhere since it eliminates needing to explicitly set the cache before grabbing stuff from it.